shuffle.py

```diff
@@ -581,19 +569,17 @@ def populate(self):
         for (dirpath, dirnames, filenames) in os.walk(self.path):
             dirnames.sort()
             # Ignore the speakable directory and any hidden directories
-            if "ipod_control/speakable" not in dirpath.lower() and "/." not in dirpath.lower():
+            if "iPod_Control/Speakable" not in dirpath and "/." not in dirpath:
```

Comment:
Actually we need to have .lower() anywhere we are assuming a directory structure inside the ipod (like here). It is perfectly valid to have either IPOD_CONTROL or iPoD_cOnTrOl and the ipod will run just fine.